packages/ui/src/components/modals/Send.tsx

```diff
@@ -20,19 +20,27 @@ import FromCallData from '../EasySetup/FromCallData'
 import { ModalCloseButton } from '../library/ModalCloseButton'
 import { formatBnBalance } from '../../utils/formatBnBalance'
 import { useGetMultisigTx } from '../../hooks/useGetMultisigTx'
+import IdentitySetIdentity from '../EasySetup/IdentitySetIdentity'
 
-const SEND_TOKEN_MENU = 'Send tokens'
-const FROM_CALL_DATA_MENU = 'From call data'
-const MANUEL_EXTRINSIC_MENU = 'Manual extrinsic'
+export const easyTransferTitle = [
+  'Send tokens',
+  'From call data',
+  'Manual extrinsic',
+  'Set identity'
+] as const
```

Comment:
Probably it's a case to enum:
enum EasyTransferTitle {
  SendTokens = 'Send tokens',
  FromCallData = 'From call data',
  ManualExtrinsic = 'Manual extrinsic',
  SetIdentity = 'Set identity',
}

Reply:
why?